src/components/02_atoms/MultipleFields/MultipleFields.js

```diff
@@ -171,13 +171,13 @@ class MultipleFields extends Component {
       state: { newItemAdded },
       props: { label, value: values, component, onChange },
     } = this;
-    const usedValues = values.length ? values : [this.createEmptyItem()];
+    const usedValues = (values && values.length) || [this.createEmptyItem()];
```

Comment:
@dawehner 
I have added the array validation also. It would make it robust. Thanks for your feedback :)

Reply:
👍 for more robustness!